Patch to eliminate dependency on pypandoc.

Patch by Marius Brehler <marbre@linux.sungazer.de>
--- setup.py
+++ setup.py.new
@@ -2,15 +2,9 @@
 #
 from distutils.core import setup
 import os
-import pypandoc
 
 from matplotlib2tikz import __version__, __license__
 
-longdesc = pypandoc.convert(
-    os.path.join(os.path.dirname(os.path.realpath(__file__)), 'README.md'),
-    'rst'
-    )
-
 setup(
     name='matplotlib2tikz',
     version=__version__,
@@ -21,7 +15,6 @@ setup(
     author_email='nico.schloemer@gmail.com',
     requires=['matplotlib (>=1.4.0)', 'numpy'],
     description='convert matplotlib figures into TikZ/PGFPlots',
-    long_description=longdesc,
     license=__license__,
     classifiers=[
         'Development Status :: 5 - Production/Stable',
